bitnami/keycloak/values.yaml

```diff
@@ -949,3 +949,12 @@ externalDatabase:
 ##
 cache:
   enabled: false
+
+## @section Keycloak Logging parameters
+
+## Keycloak logging configuration
+## ref: https://www.keycloak.org/server/logging
+## @param logging.output Alternates between the default log output format or json format
+##
+logging:
```

Comment:
Does it make sense to make it a nested parameter if only there's going to be one value?

Reply:
I'd rather call it loggingOutput and not create a section for it

Reply:
My reasoning was futureproofing it. Other users may want to add support for `log-console-format` or `log-file-format` inside the same key.

Example:
```yaml
logging:
  output: "default"
  formatFile: "%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c] (%t) %s%e%n"
  formatConsole: "%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c] (%t) %s%e%n"
```

Reply:
Fair enough